bin/update_deps.sh

```diff
@@ -47,8 +47,8 @@ go mod tidy
 sed -i "s/^BUILDER_SHA=.*\$/BUILDER_SHA=$(getSha release-builder)/" prow/release-commit.sh
 sed -i '/PROXY_REPO_SHA/,/lastStableSHA/ { s/"lastStableSHA":.*/"lastStableSHA": "'"$(getSha proxy)"'"/  }' istio.deps
 
-LATEST_DEB11_DISTROLESS_SHA256=$(docker image pull gcr.io/distroless/static-debian11 | grep Digest | awk -F\\: '{print $3}')
+LATEST_DEB11_DISTROLESS_SHA256=$(crane digest gcr.io/distroless/static-debian11 | grep Digest | awk -F\\: '{print $3}')
```

Comment:
I think the grep part is now wrong?

Reply:
Agree that on my Mac the `Digest` is now missing.

crane digest gcr.io/distroless/static-debian11 | awk -F\\: '{print $2}' seems to be correct.

Reply:
Removed the grep and updated the awk command to get rid of the warning.